### DISCUSSION TITLE
Can I deploy an express backend in one directory and then a react frontend in another onto Vercel?

### DISCUSSION THREAD

**Discussion by danblock97** [vercel/community#2736] [Help] [answered]
### Summary

My current repository (https://github.com/danblock97/apex-legends-tracker-react) consists of a backend built with Express and then a 'client' folder which is my React frontend, is there a process to get this to work in a vercel deployment? 

Currently if I deploy, I get

"404: NOT_FOUND Code: NOT_FOUND ID: lhr1::9cnqb-1685988857885-4fdfcb1f7275"

### Example

https://github.com/danblock97/apex-legends-tracker-react

### Steps to Reproduce

Clone repo and try to deploy

**Comment by amyegan** [comment, marked as answer]
Hi @danblock97! We have a guide about [using Express.js with Vercel](https://vercel.com/guides/using-express-with-vercel) and a [Turborepo Kitchen Sink Starter repo](https://github.com/vercel/turbo/tree/main/examples/kitchen-sink). Those resources will help you get your project set up appropriately for the way you want it deployed.
